nativelib/src/main/resources/scala-native/gc/commix/Phase.c

```diff
@@ -8,13 +8,13 @@
 
 void Phase_Init(Heap *heap, uint32_t initialBlockCount) {
     pid_t pid = getpid();
-    // size = static part + 32 bit int as string
-    char startWorkersName[32 + 10];
-    char startMasterName[31 + 10];
-    snprintf(startWorkersName, 32 + 10, "scalanative_commix_startWorkers_%d",
-             pid);
-    snprintf(startMasterName, 31 + 10, "scalanative_commix_startMaster_%d",
-             pid);
+    // size = static part + 10 digits (0xFFFFFFFC as maximum PID) + null char
```

Comment:
Reference for the maximum PID:
* On Windows: https://stackoverflow.com/questions/17868218/what-is-the-maximum-process-id-on-windows
* On UNIX: https://unix.stackexchange.com/questions/16883/what-is-the-maximum-value-of-the-process-id

Reply:
Great catch, thank you.  Guess that I did not throw enough 
darts at the board in my attempts to replicate the reported
failure in Issue #2200.

I do not want to do anything to slow down the merge of this PR.
With the consent of the author, I could do a follow-on PR.

I do think it is worthwhile suggesting a assert on the size
of the pid_t: i.e. that it is less than or equal to 8 bytes.
That is, guard the magic number 10 against pid_t
becoming a, say, 64 bit number.   (Or increase the
magic number to 20 (rounding up from necessary 19)
with a comment as to why 20).  Stack space is not free,
but it is cheap, especially compared to the execution time
cost of an assert. I think that SN is compiled without eliding
assertions.)

Open Group Posix 2018 says only that pid_t is a signed
integral type. Finding this bug once was hard, finding 
when someone increases the sizeof pid_t will be
hard too.  Bugs tend to re-occur.

Reply:
Given this is C code, the check for the size of pid_t could probably done with
preprocessor directives and have zero runtime cost.

Reply:
Fair point! I've widened the maximum length of the pid to 20